src/integer_mod_q/mat_zq.rs

```diff
@@ -8,6 +8,9 @@
 
 //! [`MatZq`] is a type of matrix with integer entries of arbitrary length modulo `q`.
 //! This implementation uses the [FLINT](https://flintlib.org/) library.
+//! - \[1\] John D. Dixon.
```

Comment:
use google scholar references if possible -> makes references look cleaner and more compatible throughout our project